video/out/gpu/ra.c

```diff
@@ -50,7 +50,7 @@ void ra_buf_free(struct ra *ra, struct ra_buf **buf)
 
 void ra_free(struct ra **ra)
 {
-    if (*ra)
+    if (*ra && (*ra)->fns)
```

Comment:
Might be worth splitting this into a separate patch, with a bit of context why it's needed. Seems odd to add it just now, considering there're a dozen or backends working fine w/o it.